checklists/appsvc_checklist.json

```diff
@@ -0,0 +1,304 @@
+      "subcategory": "Data Protection",
+      "text": "Use Key Vault to store TLS certificate.",
+      "description": "Store the App Service TLS certificate in Key Vault.",
+      "guid": "f8d39fda-4776-4831-9c11-5775c2ea55b4",
+      "severity": "High",
+      "link": "https://docs.microsoft.com/azure/app-service/configure-ssl-certificate"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Data Protection",
+      "text": "Isolate systems that process sensitive information",
+      "description": "Systems that process sensitive information should be isolated.  To do so, use separate App Service Plan, App Service Environments and consider the use of different subscriptions or management groups.",
+      "guid": "6ad48408-ee72-4734-a475-ba18fdbf590c",
+      "severity": "Medium",
+      "link": "https://docs.microsoft.com/azure/app-service/overview-hosting-plans"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Data Protection",
+      "text": "Do not store sensitive data on local disk",
+      "description": "Local disks on App Service are not encrypted and sensitive data should not be stored on those.  (For example: D:\\\\Local and %TMP%).",
+      "guid": "e65de8e0-3f9b-4cbd-9682-66abca264f9a",
+      "severity": "Medium",
+      "link": "https://docs.microsoft.com/azure/app-service/operating-system-functionality#file-access"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Vulnerability Management",
+      "text": "Deploy validated code",
+      "description": "Deploy trusted code that was validated and scanned for vulnerabilities according to DevSecOps practices.",
+      "guid": "19aed9c5-5d04-4c2c-9919-ca0b2c12159e",
+      "severity": "Medium",
+      "link": "https://docs.microsoft.com/azure/architecture/solution-ideas/articles/devsecops-in-azure"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Vulnerability Management",
+      "text": "Use up-to-date platforms, languages, protocols and frameworks",
+      "description": "Use the latest versions of supported platforms, programming languages, protocols, and frameworks.",
+      "guid": "114b933d-f574-4ecc-ad9b-d3bafcda3b54",
+      "severity": "High",
+      "link": "https://docs.microsoft.com/azure/app-service/overview-patch-os-runtime"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Penetration Testing",
+      "text": "Conduct a penetration test",
+      "description": "Conduct a penetration test on the web application following the penetration testing rules of engagement.",
+      "guid": "eb2eb03d-d9a2-4582-918d-2ddb10725769",
+      "severity": "Medium",
+      "link": "https://docs.microsoft.com/azure/security/fundamentals/pen-testing"
+    }
+  ],
+  "categories": [
+    {
+      "name": "Identity and Access Management"
+    },
+    {
+      "name": "Network Topology and Connectivity"
+    },
+    {
+      "name": "Business Continuity and Disaster Recovery"
```

Comment:
Do we want to add additional items for this subcategory? Would make sense to add some stuff here maybe?

Reply:
This exercise is purely focused on security recommendations for now, so not at this time - but definitely feel free to add more items in those other categories. I'd argue it should then be fairly "exhaustive" though, meaning if you would tackle the BCDR category it shouldn't be a few random thoughts but ideally a relatively "complete" set of checklist items.